src/dev/build/tasks/os_packages/docker_generator/templates/dockerfile.template.js

```diff
@@ -32,11 +32,15 @@ function generator({ artifactsUrlRoot, artifactTarball, versionTag, license  })
   
   WORKDIR /usr/share/kibana
   
+  # Copy Kibana artifact tarball inside the docker image
+  COPY --chown=1000:0 ${ artifactTarball } .
```

Comment:
Why copy the tarball here and not copy the directory the tar is created from?

Reply:
I've implemented both locally and using the tar allow us to build the docker image faster.

Time to build the default kibana docker image with tar: 1m29s
Time to build the default kibana docker image with the folder: 4m22s

Reply:
But now you are leaving the tarball in an image layer, right? This increases final image size and doesn't go over well with Docker users. We've gone to great lengths in the past to avoid this kind of bloat. [Multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/) are one way to fix this. Can I ask why you didn't preserve the [multi-stage build from the existing build system](https://github.com/elastic/kibana-docker/commit/cb432822bc5ce4b0a0c4ac2622e0b43bd1f1d2c8#diff-af46f5ebb4ebea45dc7a31805ff16956)?